index.js

```diff
@@ -25,6 +27,13 @@ module.exports.createBusClient = co.wrap(function* createBusClient(url) {
         });
       });
     }),
-    publish: (routingKey, message, options) => channel.publish(EXCHANGE, routingKey, new Buffer(JSON.stringify(message)), options)
+    publish: (routingKey, message, options) => channel.publish(EXCHANGE, routingKey, new Buffer(JSON.stringify(message)), options),
+    close: () => co(function* close() {
+      try {
+        yield channel.close();
+      } finally {
```

Comment:
maybe we should add a `catch` statement with a logger, to avoid a silent failure